tools/cluster-up/Vagrantfile

```diff
@@ -1,9 +1,12 @@
 require 'json'
 require 'net/http'
+require 'timeout'
 
 # Are we internal to TD?
 begin
-  INTERNAL = Net::HTTP.get_response("stacki-builds.labs.teradata.com", "/vagrant-boxes/").code == "200" ? "true" : "false"
+  INTERNAL = Timeout::timeout(5) {
+    Net::HTTP.get_response("stacki-builds.labs.teradata.com", "/vagrant-boxes/").code == "200" ? "true" : "false"
```

Comment:
I wonder what this does on Cox's "we hijack your unresolved dns requests and return you idiotic search results related to your company's internal dns entries".

Reply:
Probably nothing good... but I can only code around stupid so much